cerberus-audit-logger-athena/src/test/java/com/nike/cerberus/audit/logger/S3ClientFactoryTest.java

```diff
@@ -0,0 +1,16 @@
+package com.nike.cerberus.audit.logger;
+
+import com.amazonaws.services.s3.AmazonS3;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class S3ClientFactoryTest {
+
+  @Test
+  public void testS3ClientFactoryAlwaysReturnSameInstance() {
+    S3ClientFactory s3ClientFactory = new S3ClientFactory();
+    AmazonS3 s3Instance1 = s3ClientFactory.getClient("region-1");
```

Comment:
See comment above, what happens if `getClient` is called with a null or empty string?

Reply:
![image](https://user-images.githubusercontent.com/80131455/113942842-68060880-97b6-11eb-9e5f-84990241c5a0.png)
same scenario here , am i doing anything wrong ???